### DISCUSSION TITLE
Mencari capstone project di Visual Studio

### DISCUSSION THREAD

**Discussion by andhikahermawan** [teamalgoritma/community#177] [Q&A] [answered]
Halo kak, bisa dibantu saya ga menemukan capstone-flask di Visual Studio

![tanya github](https://user-images.githubusercontent.com/107476066/177811071-eeb4162e-eb14-4e66-9ef4-9a635dc56389.png)

**Comment by amaliasf** [comment, marked as answer]
Halo Pak Andhika, untuk melakukan Select Interpreter, bisa menggunakan shortcut: `Ctrl+Shift-P`. Kemudian memilih **Python Select Interpreter** . Setelah itu bisa disesuakan folder dan environment yang ingin digunakan. Semoga membantu Pak :)

**Reply by andhikahermawan** [reply]
Halo Kak, sudah menggunakan shortcut, sudah diarahkan ke kernel yang dibuat hasilnya seperti ini, apakah oke saja?
![tanya github 2](https://user-images.githubusercontent.com/107476066/177937128-f1ae0246-faee-4293-847e-32961154ba9c.png)
